cartographer_ros/cartographer_ros/node.cc

```diff
@@ -443,6 +443,16 @@ bool Node::ValidateTopicNames(
 cartographer_ros_msgs::StatusResponse Node::FinishTrajectoryUnderLock(
     const int trajectory_id) {
   cartographer_ros_msgs::StatusResponse status_response;
+
+  // First, check if we can actually finish the trajectory.
+  if (map_builder_bridge_.GetFrozenTrajectoryIds().count(trajectory_id)) {
+    const std::string error =
+        "Trajectory " + std::to_string(trajectory_id) + " is frozen.";
+    LOG(INFO) << error;
```

Comment:
Nit: LOG(INFO) << "Error string"?

Reply:
Should be LOG(ERROR), changed.

Reply:
Ah, didn't catch that. I was referring to creating a string variable - you can just stream the error message directly, without creating a variable.

Reply:
Oh yes, also true :+1:

Reply:
However, it's also true that the error string gets both logged and passed to the response, so the variable has a right to stay ;)

Reply:
Sorry, I missed that :)